Exercises.String/Exercises.String/GoogleMapsModel.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Exercises.String
+{
+    class GoogleMapsModel
+    {
+        public Dictionary<string, List<string>> UrlStruct { get; }=new Dictionary<string, List<string>>();
+
```

Comment:
Path and scheme should not be stored in same structure. It's not reliable. Also consider there can be query parameter without `value`, e.g. only with `key` (`?a&b&c`)